# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/udev/hotfix-dmi.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

WARNING: Unknown CPU family i686, please report this at https://github.com/mesonbuild/meson/issues/new

--- ./meson.build.vanilla       2025-04-08 08:40:32.213283325 +0200
+++ ./meson.build       2025-04-08 08:40:47.011145704 +0200
@@ -1537,7 +1537,7 @@
         endif
 endif
 
-dmi_arches = ['x86', 'x86_64', 'aarch64', 'arm', 'ia64', 'loongarch64', 'mips', 'riscv64']
+dmi_arches = ['x86', 'x86_64', 'i386', 'i486', 'i586', 'i686', 'aarch64', 'arm', 'ia64', 'loongarch64', 'mips', 'riscv64']
 conf.set10('HAVE_DMI', host_machine.cpu_family() in dmi_arches)
 
 # We support one or the other. If gcrypt is available, we assume it's there to


